endpointmanager/pkg/endpointmanager/postgresql/fhirendpointinfostore.go

```diff
@@ -462,6 +464,92 @@ func (s *Store) DeleteFHIREndpointInfo(ctx context.Context, e *endpointmanager.F
 	return err
 }
 
+// GetFHIREndpointInfosByURLWithDifferentRequestedVersion
```

Comment:
Should probably still have a full comment here (even if the name is pretty self explanatory)